src/main/java/org/generationcp/middleware/dao/dms/GeolocationDao.java

```diff
@@ -485,6 +486,14 @@ public Integer getNextInstanceNumber(final Integer datasetId) {
 		return 1;
 	}
 
+	public List<String> getInstanceNumbersFromIds (final List<Integer> instanceIds) {
```

Comment:
@cheryleigh Thinking about reusability, I think it will be more useful to implement getByIds, returning a list of Geolocation and then build the list out of the DAO